pkgs/build-support/rust/test/import-cargo-lock/git-dependency-workspace-inheritance/default.nix

```diff
@@ -0,0 +1,7 @@
+{ replaceWorkspaceValues, runCommand }:
```

Comment:
instead of testing the behavior of `replaceWorskpaceValues`, what if we just test `cargoLock.lockFile` directly?

```toml
[dependencies.rustpython-compiler-core]
package = "rustpython-compiler-core"
git = "https://github.com/RustPython/RustPython"
```

Reply:
We could do that, but I also kind of prefer the simplistic nature of the current test, as it's tiny + adequately tests what the tool does.

Either works -- let's wait for others to weigh in.